Makefile

```diff
@@ -51,14 +51,19 @@ lint:
 	golangci-lint --version
 	golangci-lint run --verbose --print-resources-usage --modules-download-mode=vendor --timeout=5m0s
 
+.PHONY: validate-openapi
+validate-openapi:
+	@echo "Run OpenAPI validator"
+	go run github.com/getkin/kin-openapi/cmd/validate@v0.119.0 internal/openapi/spec.yaml
```

Comment:
There is already a test that runs the `spectral` tool:

https://github.com/openshift-kni/oran-o2ims/blob/5daf9b9676c32c5248a448f5edd41c45f32735a6/internal/openapi/handler_test.go#L83-L117

If you still prefer to have this Makefile target I won't object, but it is a bit redundant. If you prefer this please remove that test.

Reply:
Oh missed this test, indeed looks good enough. I'll remove the target.